se2_planning_ros/include/se2_planning_ros/OmplReedsSheppPlannerRos.hpp

```diff
@@ -40,13 +42,22 @@ class OmplReedsSheppPlannerRos : public PlannerRos {
  private:
   void initRos();
   void publishPathNavMsgs() const;
+  void publishStartGoalMsgs(const ReedsSheppState& start, const ReedsSheppState& goal) const;
+  void initializeStateSpaceMarker();
+  void publishStateSpaceMarker();
   bool planningService(PlanningService::Request& req, PlanningService::Response& res) override;
 
   ros::Publisher pathNavMsgsPublisher_;
   ros::Publisher pathPublisher_;
+  ros::Publisher startPublisher_;
```

Comment:
can these publishers be moved to the base class? They are not specific for this particular planner.

Reply:
The base class in this case would be PlannerRos, right?

Reply:
yes, I would move them there.

Reply:
Done in 85723b842a65413c8e1fee98f81cf119f68a8b4c.